spec/gateways/sessions_spec.rb

```diff
@@ -123,4 +122,44 @@
       expect(subject.search(username: 'FOOBAR').count).to eq(0)
     end
   end
+
+  context 'recent connections' do
+    context 'within 24 hours' do
+      let(:username_1) { 'AAAAAA' }
+      let(:username_2) { 'BBBBBB' }
+      let(:username_3) { 'CCCCCC' }
+
+      it 'counts the number of successful connections' do
+        Session.create(start: today_date, success: true, username: username_1, siteIP: '127.0.0.1')
+        Session.create(start: today_date, success: false, username: username_1, siteIP: '127.0.0.2')
+        Session.create(start: today_date, success: true, username: username_2, siteIP: '127.0.0.3')
+        Session.create(start: today_date, success: false, username: username_2, siteIP: '127.0.0.4')
+        Session.create(start: today_date, success: false, username: username_2, siteIP: '127.0.0.5')
+
+        result = subject.count_distinct_users
+        expect(result).to eq(2)
+      end
+
+      it 'counts the number of unique successful connections' do
```

Comment:
It looks like this spec is testing the exact same thing as the spec above. Or am I missing something? If its not the same, it needs a better description so the next developer that comes along can see what specific scenario is being tested here.